packages/cli/test/integration.js

```diff
@@ -3675,3 +3675,79 @@ test('[vc link] should support the `--project` flag', async t => {
     formatOutput(output)
   );
 });
+
+test('vercel.json projectSettings should be applied to deployment', async t => {
+  const directory = path.join(__dirname, 'fixtures/unit/vercel-json-overrides');
+
+  const deployment = await execa(binaryPath, [
+    directory,
+    ...defaultArgs,
+    '--public',
+    '--confirm',
```

Comment:
We need on more test that doesn't use `--confirm`.

In that case, I think it should utilize vercel.json to pre-populate answers to the questions and only ask the questions that vercel.json didn't answer.

This would match how it works for the https://vercel.com/new page when importing a new git repo.